### DISCUSSION TITLE
z-push 2.6.4

### DISCUSSION THREAD

**Discussion by patois87** [Intermesh/groupoffice#847] [General]
Hello

While Group Office is using z-push 2.6.1 until now, version 2.6.4 is already released as you can see [here](https://download.kopano.io/zhub/z-push%3A/final/) (repo) and [here](https://stash.kopano.io/projects/ZHUB/repos/z-push/browse) (source).

By the way a partial changelog can be found on [this page](https://kb.kopano.io/display/ZP/Changelog).

Are there any plans to port the latest version to Group Office?

Thanks for your replies!

**Comment by slu-010101** [comment]
There is a new release (2.7.1) available and new contributors with commits:
https://github.com/Z-Hub/Z-Push/releases/

**Comment by DrDBanner** [comment]
Currently it's on 2.7.1 while 2.7.5 is out. Could be worth to get on par again:
https://github.com/Z-Hub/Z-Push/releases/
